system-contract-dapp-playground/src/utils/common/constants.ts

```diff
@@ -366,5 +366,56 @@ export const HEDERA_TRANSACTION_RESULT_STORAGE_KEYS = {
     'OWNER-OF': prepareTransactionResultStorageKey('EIP', 'ERC-721', 'OWNER-OF', true),
     'TOKEN-URI': prepareTransactionResultStorageKey('EIP', 'ERC-721', 'TOKEN-URI', true),
     'BALANCE-OF': prepareTransactionResultStorageKey('EIP', 'ERC-721', 'BALANCE-OF', true),
+    'GET-APPROVE': prepareTransactionResultStorageKey('EIP', 'ERC-721', 'GET-APPROVE', true),
+    'TOKEN-PERMISSION': prepareTransactionResultStorageKey('EIP', 'ERC-721', 'TOKEN-PERMISSION'),
+  },
+};
+
+/**
+ * @notice stores common revert reasons from wallet
+ */
+export const HEDERA_COMMON_WALLET_REVERT_REASONS = {
+  REJECT: {
+    // @notice 4001 error code is returned when a metamask wallet request is rejected by the user
+    // @notice See https://docs.metamask.io/wallet/reference/provider-api/#errors for more information on the error returned by Metamask.
+    message: '4001',
```

Comment:
This should be `code`. See https://eips.ethereum.org/EIPS/eip-1193#provider-errors 
```suggestion
    code: '4001',
```